dags/utils/gcp.py

```diff
@@ -426,6 +426,7 @@ def bigquery_etl_query(
     """
     kwargs["task_id"] = kwargs.get("task_id", destination_table)
     kwargs["name"] = kwargs.get("name", kwargs["task_id"].replace("_", "-"))
+    sql_file_path = sql_file_pathor or "sql/{}/{}.sql".format(dataset_id, destination_table)
```

Comment:
By building sql_file_path here, we avoid picking up the `$20190730` suffix that we append to `destination_table` in the next step.

Reply:
Looks like `sql_file_pathor` is a typo here and should be `sql_file_path`?

Reply:
Ah nvm, fixed in https://github.com/mozilla/telemetry-airflow/pull/573

Reply:
Yep 😭